src/script/view_model/ContentViewModel.ts

```diff
@@ -239,7 +239,7 @@ export class ContentViewModel {
       exposeMessage: exposeMessageEntity,
       openFirstSelfMention = false,
       openNotificationSettings = false,
-    } = options;
+    } = options || {};
```

Comment:
Options are not supposed to be empty, what is this change for?

Reply:
Our TS config allowed me to use this function without providing this parameter and I got an exception. And actually, in many cases we provide just `{}`, so I'd rather make `options` optional and remove those `{}`, WDYT?

Reply:
That would be the preferred way to act on this issue. If it can be undefined, it needs to be typed as such 👍

Reply:
done